1001-2000/LIT1295Dersan.xml

```diff
@@ -138,413 +146,1482 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                  </div>
+                  <div type="textpart" subtype="explicit">
+                     <ab>
+                        <l n="26">፳፮፤ ወአንትሙሂ፡ ከማሆሙ፡ ትከውኑ፡ ምዉታነ፡ ወተኀድጉ፡ ንብረተ፡ ዝንቱ፡ ዓለም፡ እስከ፡ ይእዜ፡ ኢያእመርክሙ፡ ዘእንበለ፡ ዘተደለውክሙ፡ 
+                           ወተፈጋዕክሙ፡ ተአመንክሙ፡ ንብረተ፡ ዝ፡ ዓለም፡ ኀላፊ።</l>
+                        <l n="27">፳፯፤ አንትሙሰ፡ አኃውየ፡ ኢትፍርሁ፡ ሞተ፡ ዘበምድር፡ አላ፡ ፍርሁ፡ ዘበሰማያት፡ ኀበ፡ ሀሎ፡ ኵነኔ፡ ዘነደ፡ እሳት፡ እንተ፡ አልባቲ፡ ተፍጻሜት፡ እስከ፡ 
+                           ለዓለም።</l>
+                        <l n="28">፳፰፤ ይእዜኒ፡ ሀቡ፡ ምጽዋተ፡ ለእመ፡ ብክሙ፡ ብዙኀ፡ ወለእመ፡ አልብክሙ፡ ኅዳጠ፡ በእንተ፡ ስመ፡ ቅዱስ፡ ሚካኤል፡ ወገብርኤል፡ ወይከውኑክሙ፡
+                           ረዳእያነ፡ አመ፡ ዕለተ፡ ሞትክሙ፡ በቅድመ፡ እግዚአብሔር፡ ዘሎቱ፡ ስብሐት፡ እስከ፡ ለዓለመ፡ ዓለም፡ አሜን።</l>
+                     </ab>
+                  </div>
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="MiracleBasil">
+                  <label xml:lang="en">Miracle on Basil</label>
+                  <div type="textpart" subtype="incipit">
+                     <ab>
+                        <l n="29">፳፱፤ ተአምሪሁ፡ ለመልአክ፡ ክቡር፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ዘከመ፡ ረድኦ፡ 
+                           ለ<persName ref="PRS2548Basilide">ፋሲለደስ፡</persName> ወዘከመ፡ ዐቀቦ፡ ወጾሮ፡ ዲበ፡ ክነፊሁ፡ ብርሃናውያን፡ በረከቱ፡ የሀሉ፡ ምስሌነ፡ 
+                           ለዓለመ፡ ዓለም፡ አሜን።</l>
+                        <l n="30">፴፤ ወኮነ፡ ፋሲለደስ፡ ይተግህ፡ ወይለጼሊ፡ በሌሊት፡ በድንጋፄ፡ ዐቢይ፡ ወገዓር፡ በውስተ፡ ልቡ፡ ወይፈቱ፡ ከመ፡ ይኩን፡ ሰማዕተ።</l>
+                        <l n="31">፴፩፤ ወናሁ፡ ሊቀ፡ መላእክት፡ ሚካኤል፡ አስተርአዮ፡ ሎቱ፡ በንዋም፡ በገጽ፡ ዘምሉዕ፡ ፍሥሓ፡ እንዘ፡ ይበል፡ ሰላም፡ ለከ፡ ኦ፡ 
+                           <persName ref="PRS2548Basilide">ፋሲለደስ፡</persName> ሐራሁ፡ ለክርስቶስ፡ ተፈሣሕ፡ እስመ፡ ተጽሐፈ፡ ስምከ፡ ውስተ፡ 
+                           <placeName ref="LOC3957Jerusa">ኢየሩሳሌም።</placeName>
+                           ተጽፋልና አለው። </l>
+                        <l n="32">፴፪፤ ተፈሣሕ፡ ኦ፡ <persName ref="PRS2548Basilide">ፋሲለደስ፡</persName> እስመ፡ በጽሐ፡ ምጽዋትከ፡ ኀበ፡ መንበሩ፡ 
+                           ለእግዚአብሔር።</l>
+                     </ab>
+                  </div>
+                  <div type="textpart" subtype="explicit">
+                     <ab>
+                        <l n="49">፵፱፤ ወሶቤሃ፡ ጾር፡ ወአግብኦ፡ ውስተ፡ ቤቱ፡ ወኮነ፡ ሥጋሁ፡ ብሩሀ፡ ወጼናሁ፡ ምዑዘ፡ ከመ፡ ዕፍረት፡ ወተአምኆ፡ ቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡
+                           ለቅዱስ፡ <persName ref="PRS2548Basilide">ፋሲለደስ፡</persName> ወመልኦ፡ ኀይለ፡ ወፍሥሓ፡ ወወሀቦ፡ ሰላመ፡ ወዐርገ፡ ወስተ፡ ሰማያት፡
+                           በዐቢይ፡ ስብሐት፡ ቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት።</l>
+                        <l n="50">፶፤ ትንብልናሁ፡ ወአስተብቍዖቱ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን።</l>
+                     </ab>
+                  </div>
                </div>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="CommemorativeMaskaram">
-                  <label xml:lang="en">Commemorative notice for <foreign xml:lang="gez">Maskaram</foreign> on St Michael's appearance to Isaiah
-                  </label>
+               
+               <!-- Another miracle as attested for Maskaram in <ref type="mss" corresp="BAVet82"/>: 
```

Comment:
why hidden?

Reply:
It may confuse many people to have additional content in the record, e.g. various miracles or commemorative notes for each month unless usually only one miracle and on commemorative note is needed. I recommend not to present them in parallel, because the selection of manuscript witnesses is random and not based of a indepth analysis of textual history.